### PR TITLE
[Reference] Add more stories

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Please rebase and merge (no squashing) when merging.

### CODE REVIEW

**Comment by SleeplessByte** [comment]
Dead-links tho

File: reference/stories/classes.remote-control-car.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- [C#][implementation-csharp] (reference implementation)
+
+## Related
+
+- [`concepts/object`][concepts-object]
+- [`types/class`][types-class]
+
+[concepts-object]: ../concepts/object.md
+[types-class]: ../types/class.md
+[implementation-csharp]: ../../languages/csharp/exercises/concept/class/.docs/instructions.md
```
Comment (aimorris) [inline reference/stories/classes.remote-control-car.md:38]: Dead link

Comment (aimorris) [inline reference/stories/classes.remote-control-car.md:38]: Dead link

Comment (aimorris) [reply reference/stories/classes.remote-control-car.md:38]: Should be `../../languages/csharp/exercises/concept/classes/.docs/instructions.md`

File: reference/stories/classes.remote-control-car.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## Implementations
+
+- [C#][implementation-csharp] (reference implementation)
+
+## Related
+
+- [`concepts/object`][concepts-object]
+- [`types/class`][types-class]
+
+[concepts-object]: ../concepts/object.md
```
Comment (aimorris) [inline reference/stories/classes.remote-control-car.md:36]: Dead link

Comment (aimorris) [reply reference/stories/classes.remote-control-car.md:36]: Should be `../concepts/objects.md`

File: reference/stories/classes.remote-control-car.md (hunk 1 above)
Comment (ErikSchierboom) [reply reference/stories/classes.remote-control-car.md]: Thanks!

File: reference/stories/classes.remote-control-car.md (hunk 2 above)
Comment (ErikSchierboom) [reply reference/stories/classes.remote-control-car.md]: Thanks!

**Comment by ErikSchierboom** [comment]
Thanks for the reviews!

**Pull request merged**